zebra-state/src/service/read/block.rs

```diff
@@ -88,3 +92,48 @@ where
         })
         .or_else(|| db.transaction(hash))
 }
+
+/// Returns the [`Utxo`] for [`transparent::OutPoint`], if it exists in the
+/// non-finalized `chain` or finalized `db`.
+///
+/// UTXOs may be returned regardless of whether they have been spent.
```

Comment:
Consistency note: other comments say "are".
```suggestion
/// UTXOs are returned regardless of whether they have been spent.
```

Reply:
I explained what actually happens in commit 1d4668432a.